apps/hermes/client/js/src/types.ts

```diff
@@ -0,0 +1,13 @@
+import { components } from "./serverTypes";
+
+// Accessing schema objects
+export type AssetType = components["schemas"]["AssetType"];
+export type BinaryPriceUpdate = components["schemas"]["BinaryPriceUpdate"];
+export type EncodingType = components["schemas"]["EncodingType"];
+export type GetVaaCcipInput = components["schemas"]["GetVaaCcipInput"];
+export type GetVaaCcipResponse = components["schemas"]["GetVaaCcipResponse"];
+export type GetVaaResponse = components["schemas"]["GetVaaResponse"];
+export type ParsedPriceUpdate = components["schemas"]["ParsedPriceUpdate"];
+export type PriceFeedMetadata = components["schemas"]["PriceFeedMetadata"];
+export type PriceIdInput = components["schemas"]["PriceIdInput"];
+export type PriceUpdate = components["schemas"]["PriceUpdate"];
```

Comment:
Similarly to the comment about avoiding barrel files, I also generally try to avoid `types.ts` files except in very specific scenarios.  These types nearly always are directly tied to API methods and very very often have to be imported alongside those APIs, and putting them in a separate file usually just means you have to re-export things, or the call site has to import from different places, or you have to use barrel files.  I would just move these  to the `HermesConnection.ts` file.